### PR TITLE
Add resultToString method for Select class

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

File: common/select.cpp [hunk 1]
```diff
@@ -167,4 +167,25 @@ bool Select::isQueueEmpty()
...
+            return "OBJECT";
+
+        case swss::Select::ERROR:
+            return "ERROR";
+
+        case swss::Select::TIMEOUT:
+            return "TIMEOUT";
+
+        default:
+            SWSS_LOG_WARN("unknown select result: %d", result);
```
Comment (qiluo-msft) [inline common/select.cpp]: >WARN [](start = 21, length = 4)

Treat it as an error?

File: common/select.h [hunk 2]
```diff
@@ -34,6 +34,14 @@ class Select
 
     int select(Selectable **c, int timeout = -1);
     bool isQueueEmpty();
+
+    /**
+     * @brief Result to string.
+     *
+     * Convert select operation result to string representation.
+     */
+    static std::string resultToString(int result);
```
Comment (qiluo-msft) [inline common/select.h]: >resultToString [](start = 23, length = 14)

What is the use case? Could you add a unit test for this use case?

Comment (kcudnik) [reply common/select.h]: use case is to print nice string in log if select operation fails for any reason, i will add unittests

File: common/select.cpp (hunk 1 above)
Comment (kcudnik) [reply common/select.cpp]: i set this as warning since it's not a critical issue imo

**Pull request merged**